priv/templates/phx.gen.auth/auth.ex

```diff
@@ -247,8 +256,20 @@ defmodule <%= inspect auth_module %> do
     end)
   end
 
-  <% else %>@doc """
-  Used for routes that require sudo mode.
+  @doc "Returns the path to redirect to after log in."
+  # the <%= schema.singular %> was already logged in, redirect to settings
+  def signed_in_path(%Plug.Conn{assigns: %{current_scope: %<%= inspect scope_config.scope.alias %>{<%= schema.singular %>: %<%= inspect context.alias %>.<%= inspect schema.alias %>{}}}}) do
```

Comment:
I don't understand why we have this special clause for LiveView. It feels we could get rid of it.

Reply:
From the top of my head: because we cannot store the return_to path in LiveView when navigating to the settings page and we need to reauthenticate.

For good UX, you want the user to be sent back to settings. For the non-live version this just works because we store the return_to in the sudo mode plug, but for the live version we cannot, so iirc you said when we did the magic link changes that we should just hardcore the redirect to settings when logged in for the live version.

Reply:
https://github.com/SteffenDE/phoenix_gen_auth_magic_link/pull/1#discussion_r1941436131